Source column info add (#2710)
open-cluster-management/rhacm-docs: governance/deploy_gitops.adoc

[#verifying-gitops-policy-deployments-from-the-console]
== Verifying GitOps policy deployments from the console

Verify that your changes were applied to your policies from the console. You can also make more changes to your policy from the console, however the changes are reverted when the `Subscription` is reconciled with the Git repository. Complete the following steps:

. Log in to your {product-title-short} cluster.

. From the navigation menu, select *Governance*.

> . Locate the policies that you deployed in the table. Policies that are deployed using GitOps has a _Git_ label in the _Source_ column. Click the label to be directed to the source in the Git repository.

Reviewer: ```suggestion
. Locate the policies that you deployed in the table. Policies that are deployed using GitOps have a _Git_ label in the _Source_ column. Click the label to view the details for the Git repository.
```

1. "have" goes with policies
2. It sounded like clicking the label would take me right to the repository, so I suggest a slightly different wording. But you could expand on that if you want to explicitly mention there is a link.
